lib/monitoring/checkers.go

```diff
@@ -68,8 +68,12 @@ type Config struct {
 	CloudProvider string
 	// NodeName is the kubernetes name of this node
 	NodeName string
-	// HighWatermark is the usage limit percentage of monitored directories and devicemapper
+	// HighWatermark is the usage limit percentage of devicemapper
```

Comment:
We don't have devicemapper anymore. I would also mention disk explicitly as otherwise watermark can be about anything - i.e. RAM.
Also, what is the purpose of `HighWatermark` now - is it obsolete with the introduction of these two new attributes?

Reply:
It was used for the devicemapper check. I'll go ahead and remove the devicemapper check.
Will rename warning/critical -> low/high.

Reply:
@a-palchikov 9b8e741
Okay to remove this check?